central/imageintegration/datastore/singleton.go

```diff
@@ -22,44 +23,33 @@ var (
 	ad DataStore
 )
 
-func initializeDefaultIntegrations(ctx context.Context, storage store.Store) {
-	integrations, err := ad.GetImageIntegrations(ctx, &v1.GetImageIntegrationsRequest{})
+func initializeDefaultIntegrations(storage store.Store) {
+	ctx := sac.WithGlobalAccessScopeChecker(context.Background(), sac.AllowAllAccessScopeChecker())
+	iis, err := storage.GetAll(ctx)
 	utils.CrashOnError(err)
-	if !env.OfflineModeEnv.BooleanSetting() && len(integrations) == 0 {
+	if !env.OfflineModeEnv.BooleanSetting() && len(iis) == 0 {
 		// Add default integrations
 		for _, ii := range store.DefaultImageIntegrations {
 			utils.Must(storage.Upsert(ctx, ii))
 		}
 	}
 }
 
-func initializeManager(ctx context.Context) {
```

Comment:
why is this removed?

Reply:
Yeah keeping this here meant there would be an import cycle. We determined we can move this to `enrichment/singleton.go` and it should effectively be equivalent to keeping it here

Reply:
👍